packages/www/public/sitemap.xml

```diff
@@ -1,117 +1,117 @@
-<url><loc>https://livepeer.com/blog/live-streaming-basic-workflow</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/blog/livestreaming-ecommerce-china</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/blog/livepeer-always-on-transcoding-network</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/blog/live-streaming-in-a-world-emerging-from-a-global-pandemic</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/blog/streaming-with-rtmp-api</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/jobs/1529822</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/jobs/1496262</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/jobs/1496214</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/jobs/1491881</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/jobs/1476609</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/jobs/1476601</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/jobs/1466562</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/jobs/1454194</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/jobs/1414584</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/jobs/1412804</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/jobs/1412803</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
-<url><loc>https://livepeer.com/jobs/1412799</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-08T18:27:53.187Z</lastmod></url>
+<url><loc>https://livepeer.com</loc><changefreq>daily</changefreq><priority>0.7</priority><lastmod>2022-02-09T14:28:24.457Z</lastmod></url>
```

Comment:
These changes desn't need to be committed. Just an annoying output from the www build.